.github/workflows/web.yml

```diff
@@ -76,19 +76,19 @@ jobs:
     with:
       version_file: centreon/www/install/insertBaseConf.sql
 
-  veracode-analysis:
-    needs: [get-version]
-    uses: ./.github/workflows/veracode-analysis.yml
-    with:
-      module_directory: centreon
-      module_name: centreon-web
-      major_version: ${{ needs.get-version.outputs.major_version }}
-      minor_version: ${{ needs.get-version.outputs.minor_version }}
-      stability: ${{ needs.get-version.outputs.stability }}
-    secrets:
-      veracode_api_id: ${{ secrets.VERACODE_API_ID_WEB }}
-      veracode_api_key: ${{ secrets.VERACODE_API_KEY_WEB_2 }}
-      veracode_srcclr_token: ${{ secrets.VERACODE_SRCCLR_TOKEN }}
+  # veracode-analysis:
```

Comment:
Kindly, restore the pipeline

Reply:
This is to speed up builds for testing, once ok, everything will be reactivated.